aboutMePage/aboutMePage.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="styles.css" type="text/css" />
+    <title>Joseph:Home</title>
+</head>
+<body>
+    <nav>
+        <div class="navContainer">
+            <div class="logoName">
+                JaHolt
+            </div>
+            <ul>
+                <a href=""><li>CONTACT</li></a>
+                <a href=""><li>PORTFOLIO</li></a>
+                <a href=""><li>ABOUT</li></a>
+                <a href=""><li>HOME</li></a>
+            </ul>
+            <div class="navAlignBox">
+            </div>
+        </div>
+    </nav>
+    <main>
+        <section class="titleContainer">
+            <div class="titleBox">
+                <img src="css/pics/lightbulb.png" alt="lightbulb logo" />
```

Comment:
Suggest, just making <img> without so many directories, also maybe taking out the alt as it isn't needed as long as you have the png file saved into one directory,  then in the css you can use float and padding to fill the logo to the correct position.

Reply:
I like having everything in a branching tree, I feel it is better organisation but I understand your point.